Pages/Sessions.razor

```diff
@@ -28,14 +28,27 @@
                         <thead>
                             <tr>
                                 <th>Id</th>
+                                <th>Page</th>
+                                <th>Authors</th>
+                                <th>Speaker</th>
                             </tr>
                         </thead>
                         <tbody>
                             @foreach (var lecture in session.Lectures)
                             {
                                 <tr>
                                     <td>@lecture.Id</td>
-                                </tr>
+                                    @if (@lecture is not null) {
```

Comment:
1. W `session.Lectures` nigdy nie powinno być `null`-a.
2. Jeśli byłby `null`, to i tak wykrzaczymy się w linijce wyżej na `@lecture.Id`, więc jeśli już to ten check powinien być wyżej.